packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## NEXT
```

Comment:
Should we actually do a release for this? Since the removal is emanate, users of the plugin using the latest flutter would be broken otherwise?

Reply:
(This would require updating the version number in the pubspec and mentioning it here)

Reply:
Oh, this is just in the example. Still, I think doing a minor release for this would be good.